lib/dao.js

```diff
@@ -2696,14 +2709,22 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
           inst = new Model(data, {applyDefaultValues: false});
         }
 
-        // validation required
-        inst.isValid(function(valid) {
-          if (valid) {
+        if (doValidate === false) {
+          doUpdate(where, data);
+        } else {
+          // validation required
+          inst.isValid(function(valid) {
+            if (!valid) {
+              if (doValidate) {
+                return cb(new ValidationError(inst));
+              } else {
+                g.warn('Ignoring validation errors in {{update()}}:');
```

Comment:
I think we shouldn't be printing these warnings, because they will usually make no sense. (E.g. when a model has a required "name" property, but we are updating "count" property only, then a warning about missing "name" does not make much sense.)